packages/@n8n/task-runner/src/js-task-runner/result-validation.ts

```diff
@@ -4,7 +4,19 @@ import type { INodeExecutionData } from 'n8n-workflow';
 import { ValidationError } from './errors/validation-error';
 import { isObject } from './obj-utils';
 
-export const REQUIRED_N8N_ITEM_KEYS = new Set(['json', 'binary', 'pairedItem', 'error']);
+export const REQUIRED_N8N_ITEM_KEYS = new Set([
```

Comment:
we really need to setup a `@n8n/constants` package to reduce duplication like this.